spec/openapi3.yaml

```diff
@@ -1499,7 +1499,7 @@ components:
         - rawScore
       properties:
         startHeight:
-          $ref: "#/components/schemas/Height"
+          type: string
```

Comment:
Why is this height a string a not a "schemas/Height" (long array) like the other ones?
Are we moving away from long array to string numbers?

Reply:
It is an inconsistency from the REST side, but I can confirm you that 0.7.17 returns ``activityBuckets.startHeight``  as a string.   We are [discussing](https://github.com/nemtech/catapult-rest/issues/84) right now if we should change UInt64 arrays to string numbers.

Reply:
Perfect!

IMO, I don't see the performance benefit of having high/low number style arrays. SDKs need to convert them back to u64 numbers (like BigInteger in Java), it complicates the rest APIs making them less user-friendly and it complicates SDK mappings.  I can understand that high/low can be super performant in catbuffer style serialization but not so sure in standard rest.

I also preferred the string/decimal numbers over hexadecimal numbers for the same reasons above.  Unless there is a real performance reason to do it differently, a person should be able to understand the "difficulty" by just looking at http://167.71.13.249:3000/block/1.

Let me know if you want me to test the new string number Open API spec on the Java SDK

Cheers
Fernando